news/2021-05-20-spring-fanart-contest-results.md

```diff
@@ -0,0 +1,127 @@
+
+[![](https://assets.ppy.sh/contests/125/banners/araran.jpg)](https://assets.ppy.sh/contests/125/winners/araran.png)
+
+**2767** voters succumbed to the charms of [**araran**](https://osu.ppy.sh/users/8937198)'s catgirl pippi set to a light-infused backdrop of vibrant sakura trees and clouds.
+
+With the one-two punch of gorgeous background and cute mascot all wrapped up in some serious artistic chops, it's no surprise that araran's first fanart entry ended with a spot in the finalists.
+
+[![](https://assets.ppy.sh/contests/125/banners/Jinairu[Sensei].jpg)](https://assets.ppy.sh/contests/125/winners/Jinairu[Sensei].png)
+
+**2651** voters approved of [**Jinairu[Sensei]**](https://osu.ppy.sh/users/10932731)'s take on all of the official osu! mascots (and Chirou!) lounging around in the springtime grass for a feature shot.
+
+You can sure bet that nobody is going to let Mari live down the fact that she fell asleep during this, especially Mani most of all.
+
+[![](https://assets.ppy.sh/contests/125/banners/Ayamy.jpg)](https://assets.ppy.sh/contests/125/winners/Ayamy.png)
+
+**2635** people stopped to marvel at [**Ayamy**](https://osu.ppy.sh/users/10275467)'s take on a blooming pippi undertaking what appears to be a bread run in the midst of a sunlit, springtime day.
+
+Nothing further will be mentioned, especially not the low-hanging fru--damnit.
+
+[![](https://assets.ppy.sh/contests/125/banners/ShovelKun.jpg)](https://assets.ppy.sh/contests/125/winners/ShovelKun.png)
+
+**2406** voters were taken by the hand to a land that they could understand by [**ShovelKun**](https://osu.ppy.sh/users/9948284)'s most adorable take on Mocha out on an evening springtime walk.
+
+An extra special mention goes out to the creative use of theme here - to my knowledge, nobody else delved into the evening element quite so deeply, and it really produces a stunning result here overall.
+
+---
+
+[Download a zip file containing all the winning entries in high definition here!](https://assets.ppy.sh/contests/125/winners/SpringFanart2021-Winners.zip)
+
+Be sure to check out the [Spring 2021 Fanart Contest results](https://osu.ppy.sh/community/contests/125) for the full listing of all 120+ entries to get a better picture of the entire voting.
+
+The finalist pieces described above are now available as the Spring 2021 Seasonal backgrounds, the top 3 artists have earned themselves a swanky new profile badge (of their own work, no less!) and every finalist goes home with 3 months of osu!supporter for their efforts.
+
+A huge thank you to everyone who took the time to vote - we know these contests are never easy with the sometimes overwhelming conundrum of choices to be made in the face of so many amazing entries. Another big thank you to every artist who participated too - putting your work out there like this is not easy at all, and we hope you all had fun in the process.
```

Comment:
```suggestion
A huge thank you to everyone who took their time to vote - we know these contests are never easy with the sometimes overwhelming conundrum of choices to be made in the face of so many amazing entries. Another big thank you to every artist who participated too - putting your work out there like this is not easy at all, and we hope you all had fun in the process.
```

Reply:
either works here and I think I prefer it as is